src/functions/questionnaire/create-quiz/index.ts

```diff
@@ -0,0 +1,13 @@
+import { handlerPath } from "@libs/handler-resolver";
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      httpApi: {
+        method: "post",
+        path: "/admin/questionnaire"
```

Comment:
```suggestion
        path: "/questionnaire"
```

If we are considering that only admins in the UI can create the questionnaire then we don't need to change the endpoint. We might want to do some investigation into making sure the requesting user has the correct role. Lets add a comment about checking the role in a future feature as I'm not sure about the best way to approach this right now. Probably some kind of function to decode the token and check use has the role of "admin". Please add a task with this description to the trello backlog column (at the top)